src/flow.c

```diff
@@ -344,6 +367,16 @@ void FlowHandlePacketUpdate(Flow *f, Packet *p)
         SCLogDebug("setting FLOW_NOPAYLOAD_INSPECTION flag on flow %p", f);
         DecodeSetNoPayloadInspectionFlag(p);
     }
+
+
+    /* update flow's ttl fields if needed */
+    if (PKT_IS_IPV4(p)) {
+        uint8_t ttl = IPV4_GET_IPTTL(p);
```

Comment:
can be a single line: FlowUpdateTTL(f, p, IPV4_GET_IPTTL(p));

Reply:
OK it is better with one single line.